server/server/views/registration.py

```diff
@@ -9,15 +9,30 @@
 from pyramid_mailer.message import Message
 from pyramid.response import Response
 from pyramid.httpexceptions import HTTPNotFound
+from pyramid.security import Allow, Everyone, ALL_PERMISSIONS
 
 from server.models.user import User
 from server.models.user_status import UserStatus
 from server.models.role import Role
 
+class RegistrateUser(object):
+    """Factory for adding ACL to the Service"""
 
-register = Service(name='registration', path='/registration')
+    def __init__(self, request, context=None):
+        self.request = request
+
+    def __acl__(self):
+        return [(Allow, Everyone, ALL_PERMISSIONS)]
+
+
+register = Service(name='registration',
+                   path='/registration',
+                   factory=RegistrateUser,
+                   cors_origins=('http://localhost:3000',))
 confirm_register = Service(name='email_confirm',
-                           path='/email_confirm/{email_confirm}')
+                           path='/email_confirm/{email_confirm}',
+                           factory=RegistrateUser,
+                           cors_origins=('http://localhost:3000',))
 
 
 @register.post()
```

Comment:
I think, this view seems better at user resource, in accordance to REST, post - Create.
Also I can refer to Sergiy comments at PR #15